quicktest.go

```diff
@@ -0,0 +1,106 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest
+
+import (
+	"fmt"
+	"testing"
+)
+
+// New returns a new quick test checker, that is used to call Assert and Check.
```

Comment:
s/quick test/quicktest/

There's really nothing that makes these "quick" as such :)

Also, we've got Run there too.

How about:

    // New returns a new checker instance that uses t to fail the
    // test when checks fail. It only ever calls the Assert and Check methods of t.

?

Reply:
Done, slightly differently (I guess you meant Error and Fatal rather than Assert and Check).